### PR TITLE
fix: restyle cound-badge according to vasily instructions

### PR DESCRIPTION
**Pull request by asvae**
Made them not hover:

image

### CODE REVIEW

File: src/vuestic-theme/vuestic-components/va-count-badge/VaCountBadge.demo.vue [hunk 1]
```diff
@@ -1,43 +1,43 @@
 <template>
   <VbDemo>
-    <VbCard title="0">
-      <va-count-badge :number="0">Statuses</va-count-badge>
+    <VbCard width="150px" title="0">
+      Statuses <va-count-badge :number="0"/>
     </VbCard>
-    <VbCard title="1 sign">
-      <va-count-badge :number="1">Statuses</va-count-badge>
+    <VbCard width="150px" title="1 sign">
```
Comment (jean-moldovan) [inline src/vuestic-theme/vuestic-components/va-count-badge/VaCountBadge.demo.vue]: 1 sign -> 1 digit

File: src/vuestic-theme/vuestic-components/va-count-badge/VaCountBadge.demo.vue [hunk 2]
```diff
@@ -1,43 +1,43 @@
...
+      Statuses <va-count-badge :number="1"/>
     </VbCard>
-    <VbCard title="2 signs">
-      <va-count-badge :number="12">Statuses</va-count-badge>
+    <VbCard width="150px" title="2 signs">
```
Comment (jean-moldovan) [inline src/vuestic-theme/vuestic-components/va-count-badge/VaCountBadge.demo.vue]: 2 signs -> 2 digits

File: src/vuestic-theme/vuestic-components/va-count-badge/VaCountBadge.demo.vue [hunk 3]
```diff
@@ -1,43 +1,43 @@
 <template>
   <VbDemo>
-    <VbCard title="0">
-      <va-count-badge :number="0">Statuses</va-count-badge>
+    <VbCard width="150px" title="0">
```
Comment (jean-moldovan) [inline src/vuestic-theme/vuestic-components/va-count-badge/VaCountBadge.demo.vue]: Will it be possible to add a css class:
```
.vb-card {
  width: 150px;
}
```

File: src/vuestic-theme/vuestic-components/va-count-badge/VaCountBadge.vue [hunk 4]
```diff
@@ -1,13 +1,10 @@
...
-    <span
-      class="va-count-badge__round"
-      v-if="displayNumber !== '0'"
-      :style="computedStyle"
-    >
-      {{ displayNumber }}
-    </span>
+  <span
+    class="va-count-badge"
+    v-if="displayNumber !== '0'"
```
Comment (jean-moldovan) [inline src/vuestic-theme/vuestic-components/va-count-badge/VaCountBadge.vue]: `0` is a legit number to display inside the badge. Show/hide should be handled outside of this component.

File: src/vuestic-theme/vuestic-components/va-count-badge/VaCountBadge.demo.vue (hunk 3 above)
Comment (asvae) [reply src/vuestic-theme/vuestic-components/va-count-badge/VaCountBadge.demo.vue]: @jean-moldovan I'm not following.

What's your intention?

Comment (jean-moldovan) [reply src/vuestic-theme/vuestic-components/va-count-badge/VaCountBadge.demo.vue]: Remove `width="150px"` from the template to make it a bit cleaner. Add css:
```
.vb-card {
  width: 150px;
}
```

But no problems if not.

Comment (asvae) [reply src/vuestic-theme/vuestic-components/va-count-badge/VaCountBadge.demo.vue]: @jean-moldovan it's a normal situation in demo to have cards with different sizes.
Futhermore, using css classes is discouraged in demo, inline styles are preferred.

**Comment by asvae** [comment]
Moved to vuestic-ui.

We also kept only badge. Counting is too easy to add for user, so there's no reason to keep in in framework.

**Pull request closed without merging**